app/grandchallenge/components/models.py

```diff
@@ -2080,15 +2080,23 @@ def create_civ_for_image(self, *, ci, current_civ, new_value, user):
             if created:
                 civ.full_clean()
             self.values.add(civ)
-        elif isinstance(new_value, QuerySet):
+        elif isinstance(
+            new_value, (QuerySet, UserUpload, RawImageUploadSession)
```

Comment:
For my understanding, when would we get a `user_upload` for image type interfaces?

Reply:
I thought you only get a `user_upload` for file type interfaces. If that is currently the case, I think we should not accomodate for a user_upload kwarg for image type interfaces and instead have the error raised (that you added, thanks!) so that we get notified of the change in the API.

Reply:
Ok, I thought some flexibility might be nice but makes sense to keep things simple.